Source/Fuse.Controls.Native/Fuse.Controls.Native.unoproj

```diff
@@ -2,9 +2,6 @@
   "Copyright": "Copyright (c) Fuse Open 2018",
   "Description": "Abstraction over platform-specific native controls",
   "Publisher": "Fuse Open",
-  "Package": {
```

Comment:
Whoops, this happened again. Maybe rebase? :)

Reply:
Done!